test/ValidateID.cpp

```diff
@@ -1322,7 +1322,7 @@ TEST_F(ValidateID, OpReturnValueConstantGood) {
   const char* spirv = R"(
 %1 = OpTypeVoid
 %2 = OpTypeInt 32 0
-%3 = OpTypeFunction %2 %2
```

Comment:
Not extraneous, since `OpFunctionParameter` is absent after `OpFunction`.  The old assembly was incorrect.